### ISSUE TITLE
Simple unclosed tags are flagged by no-jquery-raw-elements

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [Microsoft/tslint-microsoft-contrib#155]
`TSLint failure: Replace complex HTML strings with jQuery API: $('<div>').`

The full line is:

```typescript
html: "<pre>" + $("<div>").text(message).html() + "</pre>",
```

...so it's important to know which part is being complained about.

**Comment by HamletDRC** [comment]
This is strange because this test passes: 

    it('should pass on string literals that is not a tag', () : void => {
        const script : string = `
            $("div");
        `;

        TestHelper.assertViolations(ruleName, script, [ ]);
    });

**Comment by HamletDRC** [comment]
good find. The regex was slightly wrong
